### ISSUE TITLE
Samples from NEU

### ISSUE THREAD

**Issue by MFernflower** [OpenSourceMycetoma/Series-3-Phenothiazines#6]
I luckily came across a paper by the team at NEU about some tricyclic chlopromazine like compounds (based on carbazoles) that showed activity against Naegleria Folweri [10.3389/fmicb.2023.1149145](https://doi.org/10.3389/fmicb.2023.1149145)

Since we as a team don't have the lab capacity to make novel phenothiazines at the moment this could be a good step forward!

**Comment by MFernflower** [comment]
Compounds 8d and 8c seems promising (potent against amoeba and very close to the chlorpromazine/tricyclic antidepressant/tricyclic antihistamine scaffold):

![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/b7595e85-0097-4a66-b842-23c45dc3006f)

**Comment by lferrins** [comment]
Thanks @MFernflower, we are certainly happy to send compounds. I can prepare a list of available compounds and share with @mattodd to see what would be of interest to the project team.

**Comment by MFernflower** [comment]
My vote is for the carbazole series as published by Singh as they are very similar to chlorpromazine and other phenothiazines (that are known to have both anti-amoeba and anti-mold activity) 

as for other compounds in your library I would reach out to @mattodd or @bebi78 once your list is finished!


@lferrins

**Comment by bebi78** [comment]
@MFernflower @lferrins Compounds 8c and 8d might also be suitable for DMPK testing at Wellcome Centre in Dundee as part of the series 3 contingent since there are no other series 3 compounds selected/made.

**Comment by MFernflower** [comment]
>
> Message ID:
> ***@***.***
> com>
>
@lferrins would it be possible to make a few novel phenothiazines at NEU?
No other labs in the OSMYC group currently have time to do so and it would
be a great step forward.

the four compounds are:
N-phenethyl-phenothiazineN-isovaleroyl-phenothiazine (maybe make from
isovaleric acid anhydride?)N-isoamyl-phenothiazine N-ethyl-phenothiaIne

**Comment by MFernflower** [comment]
@lferrins @loriferrins would it be possible to make a few novel phenothiazines at NEU?
No other labs in the OSMYC group currently have time to do so and it would
be a great step forward.

**Comment by MFernflower** [comment]
Some possible structures to make at NEU:

![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/4700db3b-3bfc-488a-bda4-87a539d5bbdb)
![N-ethyl-phenothiazine](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/42f5765f-c99c-4e7a-8da6-a953cc9ec381)
![N-isoamyl-phenothiazine](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/e78fe840-61ab-4061-8fad-40d19cf1fd03)
![N-isohexyl-phenothiazine](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/b8daad89-273a-46ae-97c5-3e6c71d76143)
![N-isovaleroyl-phenothiazine](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/2641e0f6-89b8-49f2-a4d2-bef6ce7d48be)
![N-phenethyl-phenothiazine](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/b76dd506-1750-4e70-b700-286992899188)

**Comment by lferrins** [comment]
> Some possible structures to make at NEU:
> 
> ![image](https://user-images.githubusercontent.com/3164942/256016199-4700db3b-3bfc-488a-bda4-87a539d5bbdb.png) ![N-ethyl-phenothiazine](https://user-images.githubusercontent.com/3164942/256015640-42f5765f-c99c-4e7a-8da6-a953cc9ec381.png) ![N-isoamyl-phenothiazine](https://user-images.githubusercontent.com/3164942/256015642-e78fe840-61ab-4061-8fad-40d19cf1fd03.png) ![N-isohexyl-phenothiazine](https://user-images.githubusercontent.com/3164942/256015643-b8daad89-273a-46ae-97c5-3e6c71d76143.png) ![N-isovaleroyl-phenothiazine](https://user-images.githubusercontent.com/3164942/256015645-2641e0f6-89b8-49f2-a4d2-bef6ce7d48be.png) ![N-phenethyl-phenothiazine](https://user-images.githubusercontent.com/3164942/256015646-b76dd506-1750-4e70-b700-286992899188.png)

@MFernflower we don't have chemistry actively ongoing for compounds like those shown here so we won't be able to contribute synthesis at this time. However, I will circle back with what materials we have available for testing in case there is interest there.

**Comment by MFernflower** [comment]
>
> that is fine - hopefully you can build a plate up of those carbazoles and
related tricyclics you have in stock to send to Erasmus for testing


@lferrins

**Comment by MFernflower** [comment]
Worst case is if @lferrins ran out of carbazoles from them being used in the amoeba paper - @bebi78 or @wwjvdsande  could perhaps try to buy CBL0137 from a supplier? 10 mM solution in DMSO is 170 usd

**Comment by MFernflower** [comment]
> Thanks @MFernflower, we are certainly happy to send compounds. I can prepare a list of available compounds and share with @mattodd to see what would be of interest to the project team.

Any update on this? @OpenSourceMycetoma/corecontrib @lferrins

**Comment by lferrins** [comment]
@MFernflower I have sent the compound list to @mattodd and am awaiting a response. I don't have @bebi78 email address so couldn't include on the email.

**Comment by MFernflower** [comment]
@lferrins could list be uploaded to github? I do believe that Professor
Todd is on holiday at the moment.

**Comment by lferrins** [comment]
Attached is a list of all the analogs that we should have available in-house. I will preface this by saying that I have filtered out compounds that had no reported stock, or <5mg, but I have not eyeballed the vials, or run QC to confirm that they are still good. We would certainly do this if you want to test them. Please let me know and I can get folks working on it if you are interested. I am also happy to share the ADME and any potency data that we have on them.

[Curaxin_Compound_List.csv](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/files/12478155/Curaxin_Compound_List.csv)

**Comment by MFernflower** [comment]
@lferrins My vote is for the highlighted compounds + NEU-0005879 and NEU-0007251 as they are pretty close to chlorpromazine:

![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/23079490-8966-4d38-ad65-7b1def82270c)


Thoughts @OpenSourceMycetoma/corecontrib  and @bebi78 ????


Maybe also NEU-0006058 and NEU-0007251 as well

**Comment by MFernflower** [comment]
@lferrins Was curious if any updates on this project

**Comment by mattodd** [comment]
Well @MFernflower of the compounds in the list from @lferrins I would suggest any compound that gave a decent selectivity. i.e. potency better than tox (so that we're not just testing compounds that are toxic). I've not correlated the available molecules in the spreadsheet with the molecules in the tables in the publication, to identify these compounds. Could you do that @MFernflower, and come up with a final list?

**Comment by lferrins** [comment]
@mattodd I like that approach. @MFernflower I can export a new spreadsheet and filter for compounds with available stock, and HepG2 data (I think we have this for most). I'll circle back later this week.

**Comment by MFernflower** [comment]
@lferrins @mattodd I think that would be a good idea to get an CSV file of the most selective compounds - however I do think the parent compound (CBL0137) should be tested anyway since it has already been in clinical trials and is so so close in structure to things like chlorpromazine

https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/v?id=NCI-2018-01928&r=1

**Comment by MFernflower** [comment]
@mattodd I will pick from the updated list when it is uploaded

**Comment by MFernflower** [comment]
@mattodd link to previous paper where it was screened against trypanosomes https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.2c01767

**Comment by MFernflower** [comment]
Maybe we could each (Me, @bebi78  and @mattodd  pick out of a pair of compounds from Lori's library to then send to @wwjvdsande 

My vote is for the two compounds in the Singh paper:

![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/2b105bbf-453d-49df-809d-084e7b4313c1)

**Comment by lferrins** [comment]
A question, if the biological assay can handle it, why don't we just test as many as we can? How many can be run at once? Is the assay high throughput?

**Comment by MFernflower** [comment]
@lferrins you will need to talk with @wwjvdsande
<https://github.com/wwjvdsande> about the amount of different compounds
left over from the curaxin project - I am all for sending whatever you can
get 10 mg of! .

**Comment by wwjvdsande** [comment]
@lferrins @mattodd @MFernflower I usually start with a rough screening at two concentrations in a 96 well plate. So if <80 compounds I can include them in one plate. The majority in the work is afterwards to determine the IC50 and MIC50 and the larvae. We could test all of them of course, but in case they are all potent we can only select a few for the in vivo experiments next year.

**Comment by lferrins** [comment]
@wwjvdsande the number that we have available is <80 so I can work with my team to track down what we have available and get them sent to you. I will reach out to you via email to work out the logistics of that. Completely understood regarding in vivo experiments also.

**Comment by MFernflower** [comment]
44 compounds are in the list that I was given - likely gonna be less because NEU might not have stock of all of the list compounds - 5 to 10 mg of each I assume yes? @lferrins @OpenSourceMycetoma/corecontrib

**Comment by wwjvdsande** [comment]
@MFernflower @lferrins 5-10 mg stock is more than enough. The in vitro screenings will be for this winter.

**Comment by MFernflower** [comment]
@lferrins have compounds been shipped?

**Comment by lferrins** [comment]
> @lferrins have compounds been shipped?

Yes, they have been shipped, and I believe they have been received also.

**Comment by MFernflower** [comment]
Somebody will need to merge this CSV with the master list when they have time as I do not know how to do this

https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/files/12478155/Curaxin_Compound_List.csv

@OpenSourceMycetoma/corecontrib

**Comment by wwjvdsande** [comment]
@MFernflower @OpenSourceMycetoma/corecontrib we are currently testing these and the first preliminary results look promising. Noted that they are not uploaded in the masterlist yet. Can somebody do that so that we can add the activity after we have repeated the experiment?

**Comment by MFernflower** [comment]
@wwjvdsande I did upload them to a different sheet on the masterlist called "CURAXIN" - I dont know how to make new MYOS codes or how to merge the two sheets into one however

**Comment by mattodd** [comment]
OK @MFernflower done.
Can you please check that they are all in there, and then delete your Curaxin sheet?
Note that I have assigned codes based on there being no salts in this set. If there are (@lferrins), then I'll need to alter the codes.
While doing this I noticed a bunch of USyd compounds awaiting codes and which (because of a spreadsheet SNAFU) had wrong "series" values, which I've highlighted in red. Could someone please take a look @KlementineJBS @fantasy121 

@wwjvdsande the NEU codes can now be matched with MycetOS codes.

**Comment by lferrins** [comment]
@mattodd the batch number is designated by the "AA-00X" part of the molecule name and indicates that they are all the free base. Thanks for taking care of this!

**Comment by mattodd** [comment]
@lferrins yes, but I've assigned the MyOS codes on the assumption they are free bases. The NEU codes don't have that designation, so I just wanted to be sure my assumption was correct.

**Comment by lferrins** [comment]
@mattodd yeah, the original email that I sent had them. I think they might have dropped off at some point. Ok if I go and update them? It will be handy for us to associate the data in our database to the right batch

**Comment by MFernflower** [comment]
Thank you everyone!

@mattodd @wwjvdsande @lferrins 

I deleted the extra sheet and put in the notes section that the compounds were shipped as freebase

**Comment by MFernflower** [comment]
> @mattodd yeah, the original email that I sent had them. I think they might have dropped off at some point. Ok if I go and update them? It will be handy for us to associate the data in our database to the right batch



I think I ended up not importing them when I uploaded the csv to the google sheet - totally fine if you want to restore them 


@lferrins

**Comment by lferrins** [comment]
@mattodd I went through and added the batch information for the compounds that were actually shipped. Unfortunately, not all of the compounds in the spreadsheet were found/passed QC so there are more present then we actually shipped. I am not sure if you want to go through and delete the compounds that were not tested/shipped? Sorry!

**Comment by MFernflower** [comment]
Not all 44 entries in the CSV were actual real things shipped? @lferrins

**Comment by lferrins** [comment]
@MFernflower correct - the original spreadsheet had the list of compounds that we thought we had available. We had either exhausted the stocks, or they did not pass QC to be sent out. We shipped 15 compounds to Wendy in the end.

**Comment by MFernflower** [comment]
@lferrins @mattodd if you can delete the ones not sent from the master list
that would be good

I think the myos will need to be renumbered but I think I should be able to
do it

**Comment by mattodd** [comment]
So to recap someone needs to delete the added compounds with the shorter codes and keep the ones with the longer codes and then redo the MyOS codes? There should be a total of 15. When that's done someone needs to alert Wendy and Ma so that they can correlate the NEU codes with the MyOS codes and report the latter. Is that it?

**Comment by lferrins** [comment]
@mattodd yes, that is right. Sorry for the extra work.

**Comment by MFernflower** [comment]
I should be able to do that sometime today

**Comment by MFernflower** [comment]
I just did the job but only ended up with 14 entries @lferrins

**Comment by bebi78** [comment]
@MFernflower I noticed that curaxin CBL0137 was invented at Cleveland Clinic (I work together with some scientists there) as p53 activator and histone-chaperone inhibitor (FACT complex inhibitor). The compound already underwent phase 1 trial as anticancer drug (given intravenously) and was well tolerated by cancer patients (see link). So the NEU compounds are very promising drug candidates.
https://ascopubs.org/doi/abs/10.1200/JCO.2020.38.15_suppl.3583

**Comment by MFernflower** [comment]
@bebi78 I assume no data in yet for these molecules?

**Comment by MFernflower** [comment]
Interesting that pubchem has an entry for an analoge of curaxin 137 with
the diacetylcarbazole head replaced with phenothiazine   CID 61387607

**Comment by lferrins** [comment]
@MFernflower CID 61387607 is an analog of promethazine (des-methyl) and I recall that your original request for analogs featured the phenothiazine. Our focus has been on removing/modifying the acetyl groups on the CBL compounds as these could be problematic for a drug.

**Comment by MFernflower** [comment]
@lferrins there are other possible tricyclic cores to consider 
![curaxin_ideas](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/d2a6bf58-3d06-4201-b6b2-351b71622c32)

**Comment by MFernflower** [comment]
@lferrins Rather cheap too!


![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/96cff988-7241-45e6-bacf-0e9650a4159a)

**Comment by MFernflower** [comment]
> @MFernflower CID 61387607 is an analog of promethazine (des-methyl) and I recall that your original request for analogs featured the phenothiazine. Our focus has been on removing/modifying the acetyl groups on the CBL compounds as these could be problematic for a drug.

Promethazine has a different amine pattern (tertiary amine) than curaxin 137 (secondary amine) @lferrins 

It is quite interesting how similar they are however

**Comment by MFernflower** [comment]
I think a few of compounds I proposed however have the amine removed (E.G):


![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/37fbf8b7-80dc-4c09-916c-c2ae0abaf01f)

![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/62a7a9bd-ecab-4ded-b3fa-c2f8a834d78b)

@lferrins

**Comment by MFernflower** [comment]
@OpenSourceMycetoma/corecontrib I was just wondering about the status of in-vitro testing of the compounds sent from NEU

**Comment by wwjvdsande** [comment]
@MFernflower they where tested at 100 and 25 uM but need one more repetition than we will upload the data on the sheet too.

**Comment by MFernflower** [comment]
@mattodd @wwjvdsande any update on this? Is the in vitro screening pipeline backed up?

**Comment by wwjvdsande** [comment]
@MFernflower we uploaded both the high/low screening data as well as the IC50 and IC90. Apparantly there were 5 interesting compounds between them

**Comment by MFernflower** [comment]
@OpenSourceMycetoma/corecontrib could somebody post the structures of the
ones that showed interesting effects?

> Message ID:
> ***@***.***
> com>
>

@p-rayner we got some data back for a batch of tricyclics

**Comment by MFernflower** [comment]
@P-Rayner one of the NEU tricyclics came back with a good IC90 

![ketcher](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/24830bf9-8b2f-4a79-9bca-e62b658fe6c1)

Perhaps this compound could be worth making as a follow up: https://pubchem.ncbi.nlm.nih.gov/compound/61387607

**Comment by P-Rayner** [comment]
This looks really interesting data. Thanks @wwjvdsande! Hopefully some of the other tricycles will also be potent.
@MFernflower we can certainly explore some analogues as a follow up. We will update asap.

**Comment by MFernflower** [comment]
![image](https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/assets/3164942/e82fe0f4-148c-43bc-86a4-b2dd23468f4a)


@P-Rayner @lferrins since it seems like this particular tail is seeming useful - these are some possible head-swapped analogues to consider

**Comment by MFernflower** [comment]
I have also uploaded these as a CSV file that can be opened in a free tool called datawarrior

**Comment by MFernflower** [comment]
> @MFernflower CID 61387607 is an analog of promethazine (des-methyl) and I recall that your original request for analogs featured the phenothiazine. Our focus has been on removing/modifying the acetyl groups on the CBL compounds as these could be problematic for a drug.



I have posted now the few possible scaffold hops of curaxin 137 (talked about in this issue) to the series 3 wiki https://github.com/OpenSourceMycetoma/Series-3-Phenothiazines/wiki/Desirable-Compounds-Not-Yet-Synthesised @lferrins

**Comment by lferrins** [comment]
Hi all, I recall that there was a request for additional material from the NU compounds but now I am unable to locate it again, can anyone point me to it, please?

**Comment by MFernflower** [comment]
https://github.com/OpenSourceMycetoma/Monthly-zoom-meetings/issues/23#issuecomment-2188657532
@lferrins

**Comment by DouniaEMC** [comment]
In vitro data + toxicity data in vivo

| Compound | No toxicity at | IC50 | IC90 |
| :---: | :---: | :---: | :---: | 
| MYOS_00626_00_00 | 20 uM | 8.3 | 14.4 | 
| MYOS_00627_00_00 | 20 uM | 1.4 | 1.9 | 
| MYOS_00628_00_00 | 20 uM | 3.9 | 6.0 |
| MYOS_00631_00_00 | 20 uM | 2.5 | 3.7 |  
| MYOS_00632_00_00 | 20 uM | 2.4 | 3.5 |

**Comment by MFernflower** [comment]
> Hi all, I recall that there was a request for additional material from the NU compounds but now I am unable to locate it again, can anyone point me to it, please?

Any update on this? I am wondering if it would be better for Wendy's lab to order like 10 mg of curaxin 137 freebase from Sanbio to use for testing instead? 


@lferrins
